tests/test_scalars.py

```diff
@@ -1256,9 +1253,7 @@ def test_timestamp_cmp() -> None:
     c_dt_datetime = dt.datetime(year=2000, month=1, day=1)
     c_datetimeindex = pd.DatetimeIndex(["2000-1-1"])
     c_np_ndarray_dt64 = np_dt64_arr
-    c_series_dt64: pd.Series[np.datetime64] = pd.Series(
-        [1, 2, 3], dtype="datetime64[ns]"
-    )
+    c_series_dt64: TimestampSeries = pd.Series([1, 2, 3], dtype="datetime64[ns]")
```

Comment:
Is it possible to specify `dtype=np.datetime64`?

Reply:
No.  This is consistent with numpy.

```python
>>> import pandas as pd
>>> import numpy as np
>>> pd.Series([1,2,3], dtype=np.datetime64)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\series.py", line 470, in __init__
    data = sanitize_array(data, index, dtype, copy)
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\construction.py", line 622, in sanitize_array
    subarr = _try_cast(data, dtype, copy, raise_cast_failure)
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\construction.py", line 827, in _try_cast
    return maybe_cast_to_datetime(arr, dtype)
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\dtypes\cast.py", line 1321, in maybe_cast_to_datetime
    dtype = _ensure_nanosecond_dtype(dtype)
  File "C:\Anaconda3\envs\pandasstubs\lib\site-packages\pandas\core\dtypes\cast.py", line 1468, in _ensure_nanosecond_dtype
    raise ValueError(msg)
ValueError: The 'datetime64' dtype has no unit. Please pass in 'datetime64[ns]' instead.
```